src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -43,6 +44,16 @@ String user = (String) request.getSession().getAttribute("user");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+
+    // show contents of dropDown menu
+    function showDrop() {
```

Comment:
I'd make the name clearer since it works with a specific dropdown element not any dropdown.

Reply:
Also, since this switches the dropdown on and off, I'd maybe call it `toggleSettingsDropdown`?